frontend/src/actions/config.ts

```diff
@@ -1,7 +1,15 @@
-const BASE_URL =
-  process.env.REACT_APP_ENV === 'local'
-    ? 'http://127.0.0.1:8000'
-    : 'https://ervin-els-backend.herokuapp.com';
+let BASE_URL;
+
+switch (process.env.REACT_APP_ENV) {
+  case 'local':
+    BASE_URL = 'http://127.0.0.1:8000';
+    break;
+  case 'ngrok':
+    BASE_URL = 'http://fdac-49-145-101-111.ngrok.io';
+    break;
+  default:
+    BASE_URL = 'https://ervin-els-backend.herokuapp.com';
+}
```

Comment:
I have question, what do you put in REACT_APP_ENV if you are running it in local?

Reply:
I set the variable to 'local' for development and 'ngrok' when Im hosting it to ngrok.

Reply:
if its in heroku what do you put?

Reply:
on heroku the env variable is set to 'production'

Reply:
Try this one if it works
```suggestion
const env_url = {
   'local' : 'http://127.0.0.1:8000',
   'ngrok' : 'http://fdac-49-145-101-111.ngrok.io',
   'production' : 'https://ervin-els-backend.herokuapp.com'
};

const BASE_URL = env_url[process.env.REACT_APP_ENV];
```